### PR TITLE
Vacancies rss test improvement2

### PR DESCRIPTION
**Pull request by aishek**
**What this PR do?**

1. removed hardcoded strings from tests https://github.com/davydovanton/rubyjobs.dev/commit/ddb39d39ae98aa57fb25b387758612a82db8f8c6
2. add container for rss generator in controller test

### CODE REVIEW

File: apps/web/system/container.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Web app container class for dependencies
+#
+# {http://dry-rb.org/gems/dry-system/ Dry-system documentation}
+module Web
+  class Container < Dry::System::Container
+    extend Dry::System::Hanami::Resolver
+
+    use :env
+
+    register 'vacancies.generators.rss' do
```
Comment (davydovanton) [inline apps/web/system/container.rb]: it's not a good idea to create a new container for action. you can just put generator as `'web.vacancies.generators.rss'`. Could you change it?

Comment (aishek) [reply apps/web/system/container.rb]: Sure, I changed it.

**Comment by davydovanton** [comment]
could you rebase this PR and after that I'll merge it?

**Comment by aishek** [comment]
Rebased.

**Pull request merged**